package-lock.json

```diff
@@ -0,0 +1,1231 @@
+{
```

Comment:
Sorry, it's not documented, but i've used yarn and not npm. Yarn uses `yarn.lock` so this `package-lock.json` is not needed